js/bittrex.js

```diff
@@ -690,30 +690,28 @@ module.exports = class bittrex extends Exchange {
         if (body[0] === '{') {
             let response = JSON.parse (body);
             // { success: false, message: "message" }
-            if ('success' in response && 'message' in response) {
-                let success = response['success'];
-                if (typeof success === 'string')
-                    // bleutrade uses string instead of boolean
-                    success = (success === 'true') ? true : false;
-                if (!success) {
-                    const message = this.safeString (response, 'message');
-                    const feedback = this.id + ' ' + this.json (response);
-                    const exceptions = this.exceptions;
-                    if (message in exceptions)
-                        throw new exceptions[message] (feedback);
-                    if (message === 'APIKEY_INVALID') {
-                        if (this.hasAlreadyAuthenticatedSuccessfully) {
-                            throw new DDoSProtection (feedback);
-                        } else {
-                            throw new AuthenticationError (feedback);
-                        }
+            let success = this.safeValue (response, 'success');
+            if (typeof success === 'undefined')
+                throw new ExchangeError (this.id + ': malformed response: ' + this.json (response));
+            if (typeof success === 'string')
+                // bleutrade uses string instead of boolean
+                success = (success === 'true') ? true : false;
+            if (!success) {
+                const message = this.safeString (response, 'message');
+                const feedback = this.id + ' ' + this.json (response);
+                const exceptions = this.exceptions;
+                if (message in exceptions)
```

Comment:
I don't check if `message` is `undefined` because JS allows looking up `undefined` properties. Will it work in Python/PHP too?

Reply:
```
mbp:ccxt igorkroitor$ python
Python 2.7.13 (default, Jun  5 2017, 10:04:07)
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.42)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 'a' if None in {'foo':'bar'} else 'b'
'b'
```

Reply:
```PHP
php > var_dump (array_key_exists(undefined, array('foo' => 'bar')));
bool(false)
```

Reply:
Thanks! )